start.py

```diff
@@ -1,47 +1,230 @@
-def print_board(board):
-    print("-------------")
-    for row in board:
-        for elem in row:
-            print(elem, end=' ')
-        print("\n-------------")
+import random
 
+class Player:
+    def __init__(self, name):
+        self._name = name
+        self._board = None
 
-def print_available_pieces(pieces):
-    print("Available Pieces:")
-    sorted_pieces = sorted(pieces)
-    for piece in sorted_pieces:
-        print(piece, end=" ")
+    def join_board(self, board):
+        self._board = board
 
+    def place(self, piece):
+        print("Otrzymana figura: \n%s" % piece)
+        while True:
```

Comment:
ponieważ chcemy to później przerobić na interfejs graficzny. to takie wypisywanie w tym miejscu trochę nam to utrudni.
chcemy żeby wszystkie klasy pomocnicze tylko brały dane które dostarczy im klasa Game i żeby zwracały jakieś dane. nie chcemy w tych klasach robić żadnego wypisywania lub wyświetlania. tym powinna się zająć klasa Game. 

+ do tego robiąc to w taki sposób jak ja to opisałem łatwiej będzie doskonalić program, lub np. zmienić bibliotekę do grafiki  jeśli tego zechcemy

Reply:
[Zbiorcza odpowiedź](https://github.com/wikimody/Quarto-project/pull/2#issuecomment-1843002606)